scripts/messages.js

```diff
@@ -101,7 +112,11 @@ const formatMessage = (message, callback) => {
       promise: formatting.removeFromMessage(msg, assignees),
       blockers: '\n' + formatting.getStandup(message.text, action, /(blockers:|Blockers:)((.|\n)*)(last24|last 24|Last24|Last 24)/),
       last24: formatting.getStandup(message.text, action, /(last24:|last 24:|Last24:|Last 24:)((.|\n)*)(next24|next 24|Next24|Next 24)/),
-      next24: formatting.getStandup(message.text, action, /(next24:|next 24:|Next24:|Next 24:)((.|\n)*)/) + '\n'
+      next24: formatting.getStandup(message.text, action, /(next24:|next 24:|Next24:|Next 24:)((.|\n)*)/) + '\n',
```

Comment:
Why changing this logic. Think that was much better. I would even use this logic for getting points above instead of vice versa

Reply:
@zelima Which logic? logic calculating points?